lib/pbench/cli/server/session_management.py

```diff
@@ -0,0 +1,182 @@
+import logging
+
+import click
+
+from pbench import BadConfig
+from pbench.cli.server import pass_cli_context
+from pbench.cli.server.options import common_options
+from pbench.server import PbenchServerConfig
+from pbench.server.auth.exceptions import KeycloakConnectionError
+from pbench.server.auth.keycloak_admin import Admin
+
+
+# Keycloak user session cli
+@click.group("user_sessions")
+@click.version_option()
+@pass_cli_context
+@common_options
+def user_command_cli(context):
+    # Entry point
+    pass
+
+
+@user_command_cli.command()
+@pass_cli_context
+@click.option(
+    "--user_id",
+    required=False,
+    help="Keycloak user id",
+)
+@click.option(
+    "--username",
+    required=False,
+    help="Keycloak username",
+)
+@click.option(
+    "--realm",
+    required=False,
+    help="Keycloak realm name",
+)
+@click.option(
+    "--admin_token",
+    required=True,
+    help="Keycloak realm name",
```

Comment:
this help content is same as l.38. I don't know if it is intentional because it seems to be at multiple places.

Reply:
Lots of copy-and-paste going around here. 😆